src/codemagic/models/bundletool.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
+
+
+class Bundletool:
+    DOWNLOAD_URL = 'https://github.com/google/bundletool/releases/download/0.7.2/bundletool-all-0.7.2.jar'
+    DEFAULT_PATH = Path('~/programs/bundletool-all-0.7.2.jar').expanduser()
```

Comment:
Agreed. Which way do you suggest we should do deployment?

Symlink / change first and then release first cli version or search from both paths inside cli?

Reply:
Modify builder code so that it creates the symlink in initialization step in case it does not exist yet. Move `bundletool` under `/usr/local/bin` with next vm update, and default to `/usr/local/bin` in CLI from the get-go. What do you think?

Reply:
Yes, this is the better option, especially since cli-scripts will be installed by release version so we would like to minimise required backwards compatibility.
Will make the change and request new review after testing it

Reply:
Changed to `/usr/local/bin/bundletool.jar` without version